RecoMET/METPUSubtraction/plugins/DeepMETSonicProducer.cc

```diff
@@ -1,10 +1,4 @@
-#include "FWCore/Framework/interface/Event.h"
```

Comment:
@wpmccormack I think you removed a bit too much: I only asked to remove the two headers for ESHandle.h and Handle.h, which are not needed in your code. All other header includes that you removed defines objects which are used in the implementation, instead, and I thinkthey should be put back

Reply:
Hi @perrotta , ah sorry, github was making it look like the comments were aimed at multiple lines of code.  I put the headers back other than ESHandle and Handle